src/current/_includes/releases/v24.3/v24.3.0-alpha.2.md

```diff
@@ -0,0 +1,88 @@
+
+- You can now [authenticate to the DB console API]({% link v24.3/ui-overview.md %}#authentication) by supplying a Java Web Token (JWT) as a Bearer token in the Authorization header. [#130779][#130779]
+
+<h3 id="v24-3-0-alpha-2-sql-language-changes">SQL language changes</h3>
+
+- To view comments on a type, you can use the new [`SHOW TYPES WITH COMMENT`]({% link v24.3/show-types.md %}#) command. Comments can be added using [`COMMENT ON`]({% link v24.3/comment-on.md %}). [#131183][#131183]
+- You can create or alter a [user-defined function (UDF)]({% link v24.3/user-defined-functions.md %}) or [stored procedure (SP)]({% link v24.3/stored-procedures.md %}) with `[EXTERNAL] SECURITY DEFINER` instead of the default `[EXTERNAL] SECURITY INVOKER`. With `SECURITY DEFINER`, the privileges of the owner are checked when the UDF or SP is executed, rather than the privileges of the executor. The `EXTERNAL` keyword is optional and exists for SQL language conformity. [#129720][#129720]
+
+<h3 id="v24-3-0-alpha-2-operational-changes">Operational changes</h3>
+
+- The following new [metrics]({% link v24.3/metrics.md %}) show details about [replication]({% link v24.2/architecture/replication-layer.md %}) flow control send queue when the [cluster setting]({% link v24.3/cluster-settings.md %}) `kvadmission.flow_control.enabled` is set to `true` and the cluster setting `kvadmission.flow_control.mode` is set to `apply_to_all`.
+    - `kvflowcontrol.tokens.send.regular.deducted.prevent_send_queue`
+    - `kvflowcontrol.tokens.send.elastic.deducted.prevent_send_queue`
+    - `kvflowcontrol.tokens.send.elastic.deducted.force_flush_send_queue`
+    - `kvflowcontrol.range_controller.count`
+    - `kvflowcontrol.send_queue.bytes`
+    - `kvflowcontrol.send_queue.count`
+    - `kvflowcontrol.send_queue.prevent.count`
+    - `kvflowcontrol.send_queue.scheduled.deducted_bytes`
+    - `kvflowcontrol.send_queue.scheduled.force_flush`
+
+    [#131857][#131857]
+
+- The following ({% link v24.3/metrics.md %}) have been renamed:
+      Previous name                                    | New name-
+      -------------------------------------------------|-----------
+      `kvflowcontrol.tokens.eval.regular.disconnected` | `kvflowcontrol.tokens.eval.regular.returned.disconnect`
+      `kvflowcontrol.tokens.eval.elastic.disconnected` | `kvflowcontrol.tokens.eval.elastic.returned.disconnect`
+      `kvflowcontrol.tokens.send.regular.disconnected` | `kvflowcontrol.tokens.send.regular.returned.disconnect`
+      `kvflowcontrol.tokens.send.elastic.disconnected` | `kvflowcontrol.tokens.send.elastic.returned.disconnect`
+
+      [#131857][#131857]
+
+<h3 id="v24-3-0-alpha-2-db-console-changes">DB Console changes</h3>
+
+- The **Table Details** and **Tables** list in the [**Databases page**]({% link v24.3/ui-databases-page.md %}) and the table lists in **Databases** pages now use cache to reuse common data for increased efficiency [#131769][#131769]
```

Comment:
@kyle-a-wong PTAL as you reviewed the code change

Reply:
@mdlinville I think we could leave out some implementation specific detail here and leave it a bit more generalized, something like: 
```suggestion
- Revamped the UI and improved performance in the **Databases**, **Tables View**, and **Table Details** pages in the [**Databases page**]({% link v24.3/ui-databases-page.md %})  [#131769][#131769]
```

Reply:
Thanks, I edited your edit to be even a bit more concise. :)